data/json/bionics.json

```diff
@@ -588,9 +588,10 @@
     "name": { "str": "Chain Lightning" },
     "description": "Your body is equipped with a chain lightning generator, allowing you to emit a blast of lightning at a target, leaving a trail of lightning in its wake, jumping to additional targets within 4 tiles of the previous target.",
     "occupied_bodyparts": [ [ "torso", 16 ], [ "arm_l", 4 ] ],
+    "//": "Firing cost determined by item. This number is only for display.",
```

Comment:
It's a bit weird to have a value that doesn't do anything, couldn't the display code get the value from the item too instead of relying on manual input? This seem very brittle